src/journal/journald-server.c

```diff
@@ -2503,7 +2503,9 @@ int server_init(Server *s, const char *namespace) {
                 r = server_open_audit(s);
                 if (r < 0)
                         return r;
-        }
+                log_info("Audit messages are collected.");
```

Comment:
this is misleading. server_open_audit() will fail gracefully if audit support is not available. In that case it will return 0, but `s->audit_fd` will nonetheless be invalid.